babel.config.js

```diff
@@ -3,7 +3,6 @@ module.exports = {
     test: {
       presets: ['@babel/preset-env', '@babel/preset-typescript', ['@babel/preset-react']],
       plugins: [
-        '@emotion',
```

Comment:
no need?

Reply:
It breaks the spec file if this is included. We haven't needed it for a while, but it didn't really matter until I try to compare style hashes created in different files.